lms/templates/courseware/submission_history.html

```diff
@@ -1,13 +1,13 @@
-<% import json  %>
+<% import json, pytz %>
 <h3>${username | h} > ${course_id | h} > ${location | h}</h3>
 
 % for i, (entry, score) in enumerate(zip(history_entries, scores)):
 <hr/>
 <div>
-<b>#${len(history_entries) - i}</b>: ${entry.updated} (${TIME_ZONE} time)</br>
+<b>#${len(history_entries) - i}</b>: ${entry.updated.astimezone(pytz.timezone(TIME_ZONE))} (${TIME_ZONE} time)</br>
 Score: ${score.grade} / ${score.max_grade}
 <pre>
 ${json.dumps(entry.state, indent=2, sort_keys=True) | h}
 </pre>
 </div>
-% endfor
+% endfor
```

Comment:
It looks like your text editor is (mis)configured to not insert newlines at the end of files. You'll want to update that; ask if you need a hand.

GitHub tries to warn you; that's why the red arrow/circle appear at the end of line.